samples/torch/build.gradle

```diff
@@ -0,0 +1,34 @@
+apply plugin: 'konan'
+
+konan.targets = ['macbook', 'linux']
+
+def torchHome = "${System.getProperty("user.home")}/.konan/third-party/torch"
+
+task downloadTorch(type: Exec) {
+    workingDir getProjectDir()
+    commandLine './downloadTorch.sh'
+}
+
+konanArtifacts {
+    interop('TorchInterop') {
+        defFile "src/main/c_interop/torch.def"
+        includeDirs "${torchHome}/include"
```

Comment:
For me `includeDirs "${torchHome}/include", "${torchHome}/include/TH"` is needed for sample to compile.

Reply:
Fixed. (Sorry for this, I always used build.sh over build.gradle)